test/files/neg/t8700.check

```diff
@@ -0,0 +1,7 @@
+Bar.scala:2: warning: match may not be exhaustive.
```

Comment:
Argh. You are right. I somehow compared with an older file that had a package statement and was confused.